ccpp/CMakeLists.txt

```diff
@@ -96,10 +92,14 @@ endif()
 add_subdirectory(framework)
 
 add_subdirectory(physics)
-add_dependencies(ccppphys ccpp)
-target_link_libraries(ccppphys PUBLIC w3nco::w3nco_d NetCDF::NetCDF_Fortran)
-# This should not be necessary once framework and physics targets define BUILD_INTERFACE
-target_include_directories(ccppphys PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/framework/src)
+# DH*
+add_dependencies(ccpp_physics ccpp_framework)
+#
+#add_dependencies(ccppphys ccpp)
+#target_link_libraries(ccppphys PUBLIC w3nco::w3nco_d NetCDF::NetCDF_Fortran)
+## This should not be necessary once framework and physics targets define BUILD_INTERFACE
+#target_include_directories(ccppphys PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/framework/src)
+# *DH
```

Comment:
Let's remove this if it's not necessary anymore.

Reply:
I just saw a TODO comment in the PR description.

Reply:
I think I still need to add w3nco::w3nco_d somewhere in ccpp-physics's CMakeLists.txt, because the compile jobs are now failing with
```
/scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/731093042/20210910021510/ufs-weather-model/FV3/ccpp/physics/physics/sfcsub.F:8551: undefined reference to `w3kind_'
/scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/731093042/20210910021510/ufs-weather-model/FV3/ccpp/physics/physics/sfcsub.F:8554: undefined reference to `w3movdat_'
/scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/731093042/20210910021510/ufs-weather-model/FV3/ccpp/physics/physics/sfcsub.F:8599: undefined reference to `getgbh_'
...
```
@DusanJovic-NOAA Am I correct?

Reply:
Yes, add it either in ccpp_physics, in which case you'll need to 'find' w3nco package there, if you want to have ccpp physics compilable outside UFS. Or just uncomment 'target_link_library' here to define dependency of ccpp_physics target on w3nco.